requirements/base.txt

```diff
@@ -5,26 +5,26 @@
 #    make upgrade
 #
 certifi==2020.6.20        # via elasticsearch, requests
-cffi==1.14.0              # via cryptography
+cffi==1.14.2              # via cryptography
 chardet==3.0.4            # via requests
 coreapi==2.3.3            # via django-rest-swagger, openapi-codec
 coreschema==0.0.4         # via coreapi
-cryptography==2.9.2       # via pyjwt
-django-cors-headers==3.4.0  # via -r requirements/base.in
+cryptography==3.1         # via pyjwt
+django-cors-headers==3.5.0  # via -r requirements/base.in
 django-elasticsearch-dsl-drf==0.20.8  # via -r requirements/base.in
-django-elasticsearch-dsl==7.1.1  # via -r requirements/base.in, django-elasticsearch-dsl-drf
+django-elasticsearch-dsl==7.1.4  # via -r requirements/base.in, django-elasticsearch-dsl-drf
 django-nine==0.2.3        # via django-elasticsearch-dsl-drf
 django-rest-swagger==2.1.2  # via -r requirements/base.in
-django-waffle==1.0.0      # via edx-django-utils, edx-drf-extensions
-django==2.2.14            # via -c requirements/constraints.txt, -r requirements/base.in, django-cors-headers, djangorestframework, drf-jwt, edx-django-release-util, edx-django-utils, edx-drf-extensions, rest-condition
-djangorestframework==3.11.0  # via -r requirements/base.in, django-elasticsearch-dsl-drf, django-rest-swagger, drf-jwt, edx-drf-extensions, rest-condition
-drf-jwt==1.16.2           # via edx-drf-extensions
+django-waffle==2.0.0      # via edx-django-utils, edx-drf-extensions
```

Comment:
https://github.com/django-waffle/django-waffle/releases/tag/v2.0.0
Remove support for Django 1.11, 2.0, 2.1

Looks ok as long as we are actually ok on Django 2.2.16?